sdv/tabular/utils.py

```diff
@@ -155,12 +155,16 @@ def handle_sampling_error(is_tmp_file, output_file_path, sampling_error):
         raise sampling_error
 
     if is_tmp_file:
-        print('Error: Sampling terminated. Partial results are stored in a temporary file: '
-              f'{output_file_path}. This file will be overridden the next time you sample. '
-              'Please rename the file if you wish to save these results.')
+        print(  # noqa: T001
+            'Error: Sampling terminated. Partial results are stored in a temporary file: '
+            f'{output_file_path}. This file will be overridden the next time you sample. '
+            'Please rename the file if you wish to save these results.'
+        )
     elif output_file_path is not None:
-        print('Error: Sampling terminated. '
-              f'Partial results are stored in {output_file_path}.')
+        print(  # noqa: T001
+            'Error: Sampling terminated. '
+            f'Partial results are stored in {output_file_path}.'
+        )
```

Comment:
It seems like the error gets raised in the end so maybe we should just make this the error message and raise it with the error. Not sure why we print. @katxiao any specific reason for this?

Reply:
My guess is because this part is more informational, and not really related to what the error is. I'm fine with moving it to the error, as long as this statement is always shown at the top of the stack trace, so the user can see it first.

Reply:
@fealho any update on this?